acceptancetests/utility.py

```diff
@@ -291,9 +293,10 @@ def add_basic_testing_arguments(parser, using_jes=False, deadline=True,
     parser.add_argument('--keep-env', action='store_true',
                         help='Keep the Juju environment after the test'
                         ' completes.')
-    parser.add_argument('--existing', action='store',
-                        help='Existing controller to test against. Using '
-                        '"current" selects the current controller')
+    if existing:
+        parser.add_argument('--existing', action='store',
+                            help='Existing controller to test against. Using '
```

Comment:
Can the current active controller be the default here so no name needs to be passed in the simple developer use case? Basically, I'd like the following to do the right thing.

juju bootstrap lxd lxd
./assess_blah.py --existing